README.md

```diff
@@ -99,6 +99,10 @@ that meets these requirements:
 
 # Getting Started
 
+## Minimum supported rust version
```

Comment:
We can also add a badge https://img.shields.io/badge/rustc-1.46+-lightgray.svg 😁

Reply:
that's a nice idea, i'll add that.